docs/Community/sig-attestation/README.md

```diff
@@ -0,0 +1,26 @@
+Attestation Special Interest Group
+==================================
+
+SIG-Attestation is responsible for code, plugins, testing, and other works
+related to or required for attestation and integration with third-party /
+external attestation services.
```

Comment:
I think SIG-Attestation should also be responsible for tracking OESDK's alignment with attestation standards
(IETF RATS, TCG, etc.) and would like to see that stated explicitly.   I personally spend a portion of my time doing just that.

Reply:
added